awx/ui/client/src/configuration/forms/system-form/sub-forms/system-logging.form.js

```diff
@@ -75,10 +75,13 @@
                  class: 'Form-resetAll'
              },
              testLogging: {
+                 ngClass: "{'Form-button--disabled': vm.disableTestButton}",
                  ngClick: 'vm.testLogging()',
                  label: i18n._('Test'),
-                 class: 'btn-primary',
-                 ngDisabled: 'configuration_logging_template_form.$invalid'
```

Comment:
@rooftopcellist @jlmitch5 

We _could_ just ask users not to do this, but It's worth mentioning that it is possible to button mash:

![rsyslogtest](https://user-images.githubusercontent.com/9753817/76630612-b47d0700-6516-11ea-997d-9846fe4be10e.gif)

I'm guessing this is _probably_(https://github.com/ansible/awx/blob/5658c9caae1c5e6e6d763e5ca70bfbefa7200165/awx/conf/views.py#L197) safe from a backend perspective. That said, should we have any safeguards (throttling, debounce, etc) here?

Reply:
@jakemcdermott this would only be an issue in the cases where the test times out, and the UI keeps the TEST button greyed out until the view finishes processing.  However, if someone programatically hit the API endpoint for it, they might get some weird errors/behavior.

Reply:
> the UI keeps the TEST button greyed out until the view finishes processing. 

:+1: I talked about this w/ @jlmitch5 on Friday. He pushed a commit to disable the button while the promise is resolving so this LGTM.